metal/bgp.go

```diff
@@ -19,7 +18,7 @@ type bgp struct {
 	bgpPass   string
 }
 
-func newBGP(client *packngo.Client, k8sclient kubernetes.Interface, stop <-chan struct{}, project string, localASN int, bgpPass string) (*bgp, error) {
```

Comment:
stop was unused

Reply:
Yes, good catch. After the last big PR, it no longer is needed